apollo-router/src/uplink/mod.rs

```diff
@@ -437,6 +506,13 @@ mod test {
         ordering: i64,
     }
 
+    #[allow(dead_code)]
```

Comment:
Why is `allow(dead_code)` set here ?

Reply:
The fields are never read by anything other than the derived Debug implementation, which isn't enough to avoid `dead_code` warnings. Would you suggest a different approach, or a comment?

Reply:
Ah ok, it makes sense!